addons/packages/multus-cni/3.7.1/test/go.sum

```diff
@@ -1,8 +1,10 @@
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
@stmcginnis can we revert go.sum modifications to this package. While I'm sure they're correct, we should prevent this tree from being mutated by this PR.